molo/profiles/forms.py

```diff
@@ -134,7 +134,8 @@ def clean_username(self):
 class DateOfBirthForm(forms.Form):
     date_of_birth = forms.DateField(
         widget=SelectDateWidget(
-            years=list(reversed([y for y in range(1930, datetime.now().year)]))
+            years=list(reversed([y for y in range(1930,
+                                datetime.now().year + 1)]))
```

Comment:
`reversed()` takes an iterable, so you don't need the listcomp inside it. `reversed(range(a, b))` works just as well as `reversed([y for y in range(a, b)])`.